app/controllers/api/base_controller/authentication.rb

```diff
@@ -15,10 +15,9 @@ def auth_mechanism
         elsif request.headers["HTTP_AUTHORIZATION"]
           # For AJAX requests the basic auth type should be distinguished
           request.headers['X-REQUESTED-WITH'] == 'XMLHttpRequest' ? :basic_async : :basic
-        elsif request.x_csrf_token
+        elsif request.cookies['_vmdb_session'] || request.x_csrf_token
```

Comment:
I think our apps all include vmdb session, so won't this essentially turn of csrf for all browser requests?

Reply:
..how? 

Before: a cookie was ignored if csrf was not found
Now: a cookie is enough to get to the place that checks for csrf or get